docs/designers-developers/developers/README.md

```diff
@@ -26,7 +26,7 @@ You can also filter certain aspects of the editor; this is documented on the [Ed
 
 **Porting PHP meta boxes to blocks and Gutenberg plugins is highly encouraged!**
 
-Discover how [Meta Box](../../../docs/designers-developers/developers/backwards-compatibility/meta-box.md) support works in Gutenberg.
+Discover how [Meta Box](../../../docs/designers-developers/developers/backward-compatibility/meta-box.md) support works in Gutenberg.
```

Comment:
@dd32 Will changing this mean we need a redirect on the handbook environment if we change this?